components/sst_importer/src/sst_importer.rs

```diff
@@ -855,9 +861,9 @@ mod tests {
         // performs the download.
         let importer_dir = tempfile::tempdir().unwrap();
         let importer = SSTImporter::new(&importer_dir).unwrap();
-
+        let sst_writer = <TestEngine as SstExt>::SstWriterBuilder::new().build(importer.get_path(&meta).to_str().unwrap()).unwrap();
```

Comment:
This is too complex to be repeated so many times. Can you please put this into a function like `create_sample_external_sst_file_txn_default()`?